cmd/convox/builds.go

```diff
@@ -623,6 +616,7 @@ func createTarball(base string) ([]byte, error) {
 }
 
 func finishBuild(c *cli.Context, app string, build *client.Build) (string, error) {
+	fmt.Println("Streaming build logs...")
```

Comment:
Will do. I liked the print entry so we could know where in build process the CLI was. But I'm OK either way.